sdcm/utils/loader_utils.py

```diff
@@ -117,6 +117,47 @@ def _run_all_stress_cmds(self, stress_queue, params):
 
         return stress_queue
 
+    @staticmethod
+    def parse_cs_user_profiles_param(cs_user_profiles):
+        user_profiles_def = list(zip(*[profile.split(",") for profile in cs_user_profiles]))
+        user_profiles = user_profiles_def[0]
+        durations = None if len(user_profiles_def) == 1 else user_profiles_def[1]
+        return user_profiles, durations
+
+    def run_cs_user_profiles(self, cs_profiles: str | int | list, duration: str | list = None, stress_queue: list = None):  # pylint: disable=too-many-locals
+        stress_queue = stress_queue or []
+        stress_cmds = []
+        if not isinstance(cs_profiles, list):
+            cs_profiles = [cs_profiles]
+
+        if duration and not isinstance(duration, list):
+            duration = [duration]
+
+        round_robin = self.params.get('round_robin')
+        for i, cs_profile in enumerate(cs_profiles):
+            assert os.path.exists(cs_profile), 'File not found: {}'.format(cs_profile)
+            self.log.debug('Run stress with user profile %s', cs_profile)
+            profile_dst = os.path.join('/tmp', os.path.basename(cs_profile))
+            with open(cs_profile, encoding="utf-8") as file:
+                content = file.readlines()
+                for cmd in [line.lstrip('#').strip() for line in content if line.find('cassandra-stress') > 0]:
+                    stress_cmd = (cmd.format(profile_dst)) if duration is None else (
+                        cmd.format(profile_dst, duration[i].strip()))
+                    params = {'stress_cmd': stress_cmd, 'profile': cs_profile, 'round_robin': round_robin}
+                    stress_params = dict(params)
+                    user = self.params.get('authenticator_user')
+                    password = self.params.get('authenticator_password')
+                    if user and password and 'user=' not in stress_cmd:
+                        # put the credentials into the right place into -mode section
+                        stress_cmd = re.sub(
+                            r'(-mode.*?)-', r'\1 user={} password={} -'.format(user, password), stress_cmd)
+
+                    self.log.debug('stress cmd: {}'.format(stress_cmd))
+                    stress_cmds.append(stress_cmd)
+                    stress_queue.append(self.run_stress_thread(**stress_params))
+
+        return stress_queue, stress_cmds
```

Comment:
as I've seen the stress_commands are being used only to get the profile locations, let change this return value to be the proper profile locations, and not the whole stress command, which the user of this function just gave, he doesn't need it back.

Reply:
Profile location is different on runner and on loader. `run_cs_user_profiles` is performed on loader. Next usage of user profile may be on different location

Reply:
removed